util/settings/accounts.go

```diff
@@ -307,7 +307,7 @@ func parseAccounts(secret *v1.Secret, cm *v1.ConfigMap) (map[string]Account, err
 			account.Tokens = make([]Token, 0)
 			if string(tokensStr) != "" {
 				if err := json.Unmarshal(tokensStr, &account.Tokens); err != nil {
-					return nil, err
+					log.Errorf("Invalid token '%s' found in secret '%s'", string(tokensStr), secret.Name)
```

Comment:
The token might be invalid because it has trailing whitespace characters. In this case, we might log information that contains a valid token which is a security issue. Just to be safe lets print the only account name:

```
log.Errorf("Account '%s' has invalid token in secret '%s'", name, secret.Name)
```